### ISSUE TITLE
Incorrect heading usage (missing H2s, skipped to H3s)

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/justgetsimple#61]
For example: Our Mission, Our Organization, and Our Vision all use H3s when it should be using H2s.

Events also use incorrect headings—event titles should ideally have H4s.
